pennylane/templates/subroutines/aqft.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This submodule contains the template for AQFT.
+"""
+# pylint:disable=abstract-method,arguments-differ,protected-access
+
+import warnings
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import Operation
+
+
+class AQFT(Operation):
+    r"""AQFT(order, wires)
+    Apply an approximate quantum Fourier transform (AQFT).
```

Comment:
Could you add a Methematical formula or link the paper which describes the method for the AQFT?